REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.4
 Nettle 0.2
 JSON 0.5
-ZMQ 0.3.1
+ZMQ 0.3.4
```

Comment:
I mean, for Julia 0.4, ZMQ 0.3.1 was defined for the type `Message` with
```
bytestring(zmsg::Message) = bytestring(pointer(zmsg), length(zmsg))
```
and with this PR, calling `unsafe_string(::Message)` doesn't work on Julia 0.4.

when running jupyter-notebook, there's hanging with timeout like this
```
[W 21:35:58.097 NotebookApp] Timeout waiting for kernel_info reply from e3f8da4c-ce1c-4604-965e-200479992642
```

thanks.